test/model-inheritance.test.js

```diff
@@ -258,7 +259,7 @@ describe('Model class inheritance', function() {
       });
 
       assert.deepEqual(Customer.settings, {
-        forceId: true,
+        forceId: false,
```

Comment:
Since the Customer has specified `customerId: {type: String, id: true}` as a non-generated property, `forceId` it makes sense to me that `forceId` should be disabled, even though the parent `User` model has it enabled. This is the behaviour we are relying on in LoopBack, isn't it?